src/config.ts

```diff
@@ -1,24 +1,19 @@
 import { ConfigExtender } from "hardhat/types";
 
-export const deployConfigExtender: ConfigExtender = (resolvedConfig, config) => {
+export const getDefaultGoBindConfig: ConfigExtender = (resolvedConfig, config) => {
   const defaultConfig = {
-    from: -1,
-    to: -1,
-    only: -1,
-    skip: -1,
-    confirmations: 0,
-    verify: false,
-    pathToMigrations: "./deploy",
-    skipVerificationErrors: ["already verified"],
-    verificationAttempts: 1,
+    outDir: './artifacts/gobind',
+    deployable: false,
+    runOnCompile: false,
+    abigenPath: './node_modules/hardhat-gobind/bin/abigen'
```

Comment:
This internal variable should not be preset in the user config

Reply:
I was thinking about using this in the environment. The problem is that user may either have Windows (so we'll need to check his OS additionally), or wish to build his own binary because the given one won't be suitable.

Reply:
Actually, let's not include the actual binary in the plugin. We will ask the user to preinstall it before the plugin installation

Reply:
In that case, the parament in the config might be relevant. However, the default value should be `""` and the binary path resolver should try to use the `abigen` from the shell directly